dot/network/discovery.go

```diff
@@ -92,6 +93,9 @@ func (d *discovery) waitForPeers() (peers []peer.AddrInfo, err error) {
 
 // start creates the DHT.
 func (d *discovery) start() error {
+	// this basically only works with enabled mDNS which is used only for local test setups. Without bootnodes kademilia
+	// would not bee able to connect to any peers and mDNS is used to find peers in local network.
+	// TODO: this is an ugly solution and needs to be refactored
```

Comment:
I believe that this refactor can be in this PR, given that we don't need to wait for peers to start the discovery